content/about/index.md

```diff
@@ -10,4 +10,33 @@ Packit provides tooling and automation to integrate upstream open source project
 
 https://github.com/packit-service/packit
 
+
+### The key principles of packit
+
+* Reuse existing tools and services where it makes sense: don't reinvent the
+  wheel.
+
+* You can consume packit in two forms:
+  * Packit tool can run on your laptop and you run commands you want.
+  * Packit service reacts to events and performs actions which you
+    defined in your `packit.yaml`.
+
+* We don't break backwards compatibility just like that.
+
+* Packit has a deprecation policy:
+  * We can mark a behaviour as deprecated.
+  * Deprecated content can be removed or changed after at least 3 minor
+    releases.
+  * Deprecated content is advertised in our
+    [changelog](https://github.com/packit-service/packit/blob/master/CHANGELOG.md),
+    printed in terminal or tracked in your issue tracker.
+
+* We care about artifacts which Fedora supports: at the moment it's RPMs,
+  modules and container images.
+  * Our initial focus is solely on RPMs.
+
+* Packit respects Fedora guidelines.
+
+* We want latest content in Fedora Rawhide, but only if it works.
+
 <!--more-->
```

Comment:
Does it make sense to have the `<!--more-->` at the very bottom of the page?

Reply:
I want the principles to be displayed on the landing page. And probably add more content to the about page.